cmd/go-clang-generate/function.go

```diff
@@ -16,10 +16,14 @@ type Function struct {
 	Parameters          []FunctionParameter
 	ReturnType          string
 	ReturnPrimitiveType string
+	IsReturnTypePointer bool
+	IsReturnTypeArray   bool
```

Comment:
convert the "ReturnType" member into a "Conversion" because it has almost all of the "Return*" members in it.

Also, how about renaming "Conversion" to just "Type" ?